Description: This patch corrects some upstream spelling errors and typos.
Author: Ignace Mouzannar <mouzannar@gmail.com>
Last-Update: 2011-11-16
--- a/doc/tsplot.1
+++ b/doc/tsplot.1
@@ -1,4 +1,4 @@
-.\\" auto-generated by docbook2man-spec $Revision: 1.1 $
+.\" auto-generated by docbook2man-spec $Revision: 1.1 $
 .TH "TSPLOT" "1" "February 2007" "" ""
 .SH NAME
 tsplot \- Plot several tsung logs on the same charts, for comparison purpose.
@@ -34,7 +34,7 @@
 what it does.
 .SH "CONFIGURATION"
 .PP
-The configuration file of \fBtsplot\fR allows to
+The configuration file of \fBtsplot\fR allows one to
 define the plots you want to obtain, from their label to the
 data they will show. The configuration file adopts a
 \fI\&.ini\fR file syntax, each section defining a
@@ -47,7 +47,7 @@
 and a \fBtsung\fR PGSQL test.
 .PP
 A \fBDEFAULT\fR section may be provided, any
-element configured here may then be overriden into a specific
+element configured here may then be overridden into a specific
 plot section.
 .PP
 Another configuration file is used by \fBtsplot\fR:
--- a/doc/tsung-recorder.1
+++ b/doc/tsung-recorder.1
@@ -17,7 +17,7 @@
 tsung native XML format; it can be used by your favorite client (browser in the case of the http plugin). 
 .TP
 \fBstart\fR
-start the proxy recorder (listening port is 8090). By default the HTTP recorder is started. With the -p option, you can select another plugin. The resulting files will be created as \fI~/.tsung/tsung_recorderYYYMMDD-HH:MM.xml\fR; if it doesn't work, take a look at \fI~/.tsung/log/tsung.log-tsunami_recorder@hostname\fR 
+start the proxy recorder (listening port is 8090). By default the HTTP recorder is started. With the \-p option, you can select another plugin. The resulting files will be created as \fI~/.tsung/tsung_recorderYYYMMDD-HH:MM.xml\fR; if it doesn't work, take a look at \fI~/.tsung/log/tsung.log-tsunami_recorder@hostname\fR 
 .TP
 \fBstop\fR
 stop the proxy recorder
